src/components/transfers/TransferTypes.ts

```diff
@@ -0,0 +1,14 @@
+export const TransferStatus = {
+  initial: 'initial',
+  invalid: 'invalid',
+  incomplete: 'incomplete',
+  declined: 'declined',
+  cancelled: 'cancelled',
+  succeeded: 'succeeded',
+}
+
+export const Currency = {
+  BGN: 'BGN',
+  EUR: 'EUR',
+  USD: 'USD',
+}
```

Comment:
We already have `Currency` defined in another file

Please reuse it instead of adding an additional one